src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -780,6 +780,26 @@ default boolean collectionSendImage() {
         return true;
     }
 
+    @ConfigItem(
+        keyName = "collectionDenylist",
+        name = "Item Denylist",
+        description = "Never fire notifications for these items.<br/>" +
+            "Place one item name per line (case-insensitive; asterisks are wildcards)",
+        position = 3,
+        section = collectionSection
+    )
+    default String collectionDenylist() {
+        return "Purple sweets\n" +
```

Comment:
I don't think we should include a denylist by default - we could include these but commented out? (assuming we support comments here)

I don't want to get into a scenario where someone is using this to track their progress on things (i.e. a non-discord use case) and these don't fire after an update, but fired before

Reply:
Technically we don't support comments, but we could put `# ` in front of each, which would never match in practice (and tell the user to uncomment lines they want)